pkg/shell/active-pages-modal.jsx

```diff
@@ -110,3 +113,5 @@ export class ActivePagesDialog extends React.Component {
         );
     }
 }
+
+ActivePagesDialog.contextType = DialogsContext;
```

Comment:
Oh, this is nice. Does ESLint or React warnings in our /devel scenario enforce this?

Reply:
I don't think so. I missed it a couple of times, and then `this.context` remains `undefined`.